libraries/package-deps-hash/src/getPackageDeps.ts

```diff
@@ -10,20 +10,26 @@ export function parseGitLsTree(output: string): Map<string, string> {
   const changes: Map<string, string> = new Map<string, string>();
 
   if (output) {
+    // A line is expected to look like:
+    // 100644 blob 3451bccdc831cb43d7a70ed8e628dcf9c7f888c8    src/typings/tsd.d.ts
+    // 160000 commit c5880bf5b0c6c1f2e2c43c95beeb8f0a808e8bac  web-build-tools
+    const gitRegex: RegExp = /([0-9]{6})\s(blob|commit)\s([a-f0-9]{40})\s*(.*)/;
```

Comment:
Yes
The first 3 digits specify the type
Then the next 3 digits are unix file modes (r/w/x)
https://stackoverflow.com/questions/737673/how-to-read-the-mode-field-of-git-ls-trees-output/8347325#8347325